### PR TITLE
Add erb report, 2 generators

### PR DESCRIPTION
**Pull request by WiltonPyle**

### CODE REVIEW

File: lib/generators/icg.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+
+# This generates a pseudo-random sequence by using the an inverse
+# congruential generation method
+
+require_relative '../generator.rb'
+require 'prime'
```
Comment (kedji) [inline lib/generators/icg.rb]: What is this require used for?

File: lib/generators/pruned_bit.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    desc = <<-DESC_END
+      This test shows that the statistical tests easily catch something like
+      a particular bit that appears <1% less often than its counterparts.
+      The binomial test is particularly sensitive to this case.
+    DESC_END
+    desc.gsub(/\s+/, " ").strip
+  end
+
+  def next_chunk
+    @i += 1
```
Comment (kedji) [inline lib/generators/pruned_bit.rb:32]: It looks like `@i` is incremented but never used.

File: lib/generators/pruned_bit.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    @s = (@s * 6364136223846793005 + 1442695040888963407) % 2**64
+    raw = Digest::MD5.digest(@s.to_s)
+    #prune LSB randomly. about half the expected effect since some are already zero
+    raw = raw.unpack('C*').map { |x| rand < 0.01 ? x & 0xFE : x }.pack('C*')
```
Comment (kedji) [inline lib/generators/pruned_bit.rb]: The use of `rand` introduces non-determinism between runs. What would you think about using `@i` (incrementing inside the `map`) to reset every 100th LSB?

**Comment by WiltonPyle** [comment]
Updated these 3 issues in the last commit. The require 'prime' was left over from a slower implementation. Now making use of the @i instead of rand to keep it deterministic in pruned_bit.

**Pull request merged**